assets/js/modules/analytics-4/datastore/properties.test.js

```diff
@@ -543,20 +543,23 @@ describe( 'modules/analytics-4 properties', () => {
 			} );
 		} );
 		describe( 'updateSettingsForMeasurementID', () => {
-			it( 'should update the settings with the measurement ID.', () => {
+			it( 'should update the settings with the measurement ID.', async () => {
```

Comment:
The changes here look like more than they really are, but it updates the assertions to be a bit cleaner but also more consistent with how we should be testing actions that mutate state, by checking the state directly rather than by our own selector which have their own tests.